Console/Command/Indexer/ProcessQueueCommand.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Console\Command\Indexer;
+
+use Algolia\AlgoliaSearch\Helper\ConfigHelper;
+use Algolia\AlgoliaSearch\Model\Queue;
+use Algolia\AlgoliaSearch\Service\AlgoliaCredentialsManager;
+use Magento\Framework\App\Area;
+use Magento\Framework\App\State;
+use Magento\Framework\Console\Cli;
+use Magento\Framework\Exception\LocalizedException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ProcessQueueCommand extends Command
+{
+    protected ?OutputInterface $output = null;
+
+    public function __construct(
+        protected State $state,
+        protected ConfigHelper $configHelper,
+        protected Queue $queue,
+        protected AlgoliaCredentialsManager $algoliaCredentialsManager,
+        ?string $name = null
+    ) {
+        parent::__construct($name);
+    }
+
+    protected function getCommandName(): string
+    {
+        return 'algolia:reindex:process_queue';
```

Comment:
This change will impact our [current guidelines](https://www.algolia.com/doc/integration/magento-2/how-it-works/indexing-queue/?client=php#process-the-queue). I'm thinking rather than just pointing to a new CLI, this is a good opportunity to finally abandon the explicit separate crontab and instead use Magento's `crontab.xml` setup. We have a task for this in the backlog.

Reply:
I think that this is still better to have a "non Magento indexer" command for this, because we want to make sure that this isn't processed on setup:upgrade for example. I don't think that creating this would be bad but I'm ok to explore the `crontab.xml` path but it's scheduled for Phase 2. 

In the meantime, maybe `algolia_queue_runner` indexer shouldn't be disabled by default to avoid breaking changes on upgrade for users who already have setup their cron as in the doc ?